media/pages/sk/ochrana-osobnych-udajov/page.html

```diff
@@ -55,8 +59,9 @@ <h3>
 <p>
 <span>2.1.</span>
 Prevádzkovateľ spracúva osobné údaje dotknutej osoby v informačnom systéme chcemvediet.sk na účely
-sprostredkovania podávania žiadostí o informácie v zmysle zákona č. 211/2000 Z.z. o slobodnom
-prístupe k informáciám prostredníctvom portálu www.chcemvediet.sk. 
+sprostredkovania podávania žiadostí o informácie v zmysle zákona č.
+<a href="https://www.slov-lex.sk/pravne-predpisy/SK/ZZ/2000/211/" target="_blank">211/2000 Z.z.</a>
+o slobodnom prístupe k informáciám prostredníctvom portálu www.chcemvediet.sk.
```

Comment:
@martinmacko47 niektore slovne spojenia typu "zakon o slobodnom ..." som nahradil medzeru s `NBSP` teda "zakon o\NBSPslobodnom ...". Takisto by som to spravil aj v `podmienky-pouzivania/page.html` uz v samostatnom PR.

Reply:
@viliambalaz Ano, typografiu a trvde medzery kludne poopravuj vsade, kde si vsimnes, ze je chyba.